### DISCUSSION TITLE
Use token instead of username and password during re authentication

### DISCUSSION THREAD

**Discussion by praneeth100** [strongswan/strongswan#2424] [General]
For IKEv1 or IKEv2 remote access clients. Is there a plugin(or standard) to send token to the client after successful authentication so that re-authentication happens using token instead of username/password? 
It would be useful for OTP based authentications where re-auth fails due to OTP expiry.

**Comment by tobiasbrunner** [comment]
Possibly [RFC 5723](https://datatracker.ietf.org/doc/html/rfc5723) (currently not implemented).
